bin/yb-ctl

```diff
@@ -711,8 +711,7 @@ class ClusterOptions:
 
         # -----------------------------------------------------------------------------------------
         # Automatic YugaByte DB installation
-        # -----------------------------------------------------------------------------------------
-
+        # -----------------------------------------------------------------------------------------  
```

Comment:
> bin/yb-ctl:714:100: W291 trailing whitespace
can you undo this line? I think it broke our CI

Reply:
done